app/javascript/dashboard/store/modules/auth.js

```diff
@@ -118,6 +118,16 @@ export const actions = {
     }
   },
 
+  deleteAvatar: async ({ commit }) => {
+    // eslint-disable-next-line no-useless-catch
```

Comment:
We could avoid this by `throw new Error(error.message);` or `throw new Error(error);`

Reply:
changed